test/export/test_export.py

```diff
@@ -75,5 +79,46 @@ def forward(self, x):
         # self.assertEqual(mutated_buffer.sum().item(), 30)
         self.assertEqual(output, mod(*inp))
 
+    @unittest.skip("RuntimeError: Python 3.11+ not yet supported for torch.compile")
+    @unittest.skip("RuntimeError: Windows not yet supported for torch.compile")
+    @config.patch(dynamic_shapes=True, capture_dynamic_output_shape_ops=True, specialize_int=True, capture_scalar_outputs=True)
+    def test_export_constraints(self):
+
+        def f(x):
+            b = x.item()
+            add_inline_size_constraint(b, min=2, max=5)
+            return torch.full((b, 1), 1)
+
+        inp = (torch.tensor([3]),)
+        ref = f(*inp)
+
+        gm, _ = torchdynamo.export(f, *inp, aten_graph=True, tracing_mode="symbolic")
+        res = gm(*inp)
+
+        self.assertTrue(torchdynamo.utils.same(ref, res))
+
+        gm = make_fx(f, tracing_mode="symbolic")(*inp)
+        res = gm(*inp)
+        self.assertTrue(torchdynamo.utils.same(ref, res))
+
+    @unittest.skip("RuntimeError: Python 3.11+ not yet supported for torch.compile")
```

Comment:
Can you make it skipIf? I think i saw some utility functions in dynamo about skipping if python 3.11.